src/app/component/layout/components/header/header.component.scss

```diff
@@ -532,6 +538,11 @@ $screen-320: " screen and (max-width : 320px)";
 }
 
 @media #{$screen-576} {
+  .logo-img {
```

Comment:
If to be honest, all breakepoints for media query and max-width property must be one pixel less, for example 319, 575, 767 etc. due to mockup requirements